Menu/MenuManagerInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Menu;
+
+/**
+ * @author Christian Gripp <mail@core23.de>
+ */
+interface MenuManagerInterface
```

Comment:
Please refrain from using vague things like manager or service in class names. MenuCollection or MenuRegistry might be better in this case: if you can't find a more precise name, people will start adding all sorts of unrelated things in the class and you will end with many SRP violations.